scripts/fetch-sapling-params.sh

```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/env bash
+# SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+# This script is an alternative for
+# https://raw.githubusercontent.com/zcash/zcash/master/zcutil/fetch-params.sh.
+# This script downloads only `sapling-*.params` files to the `/usr/local/share/zcash-params`,
+# so that the downloaded files can be used by different users. Since it downloads files
+# to `/usr/local/share`, it should be run with sudo.
+set -euo pipefail
+
```

Comment:
Since `sudo` is pretty much necessary, how about adding this for a better error message?
```suggestion

if [ "$EUID" -ne 0 ]; then 
        echo "This script requires root privileges."
        echo "Please run it again with sudo or as root."
        exit 1
fi

```

Reply:
I think this suggestion is what caused the trailing whitespaces check to fail (line 13).

@rvem  Can you check if that's the case and fix it?

Reply:
This script should be useless after #132